### ISSUE TITLE
Car Poofing

### ISSUE THREAD

**Issue by Daemon53** [DSRP-Inc/DSRP_Tickets#32]
on this I saw a GitHub script that prevents car poofing

**Comment by IngleArf** [comment]
was this a report? i do have the script but do not want to add it unless its needed

**Comment by Daemon53** [comment]
Well now I'm not too sure

**Comment by IngleArf** [comment]
put the script in

It saves cars position every 10 seconds, checks every 15 seconds, and saves position on enter and exit of vehicle

Player owned cars should not despawn anymore when you get too far away or disconnect from the server due to head pop

**Comment by shelby0341** [comment]
can confirm script seems to be working as intended.
owned a PDM "Truffade Adder" took it to the lumber yard in Paleto Forest, teleported to the docks and back and it was still there.
drove a local car from the Adder in Paleto to Little Seoul and back and it was still there.
disconnected from the server and reconnected and it was still there.
moving to completed.

**Comment by shelby0341** [comment]
This is no longer working, player vehicles are despawning when you leave the area.

**Comment by IngleArf** [comment]
This is ready for testing

in my testing the only thing that made a car poof was a server restart

Needs to be tested before next change log

**Comment by IngleArf** [comment]
Tested:

Player owned cars, parked at grove, drove to sandy, and then back, car still there

Rented Vehicles: parked at airport lot, drove to sandy in spawned car, drove back, car still there.


Did same for both except no clipped to paleto and back, cars still there

Local cars will continue to despawn due to the fact that the game itself controls this, it would tank performance otherwise

**Comment by shelby0341** [comment]
Player owned vehicles are still being reported as poofing
Discord ticket: https://discord.com/channels/975465175178153994/1160340663666212886
"This has happened at every construction location and even some house robberies when I leave the house. This is just one example but it's been happening just about every other construction job which I have to pay the $500 to get it back out so it's making it hard to progress. It doesn't happen every time. So it seems like it's when nobody is in the area is when it poofs."

**Comment by IngleArf** [comment]
Yes. I believe it's because in the clip, he entered the van so the game prioritized the vehicle he was driving instead of the one he had taken out previously. 

Temporary stop gap could be to add garages to those places

There is a paid script for this and it's 22 bucks

**Comment by shelby0341** [comment]
@IngleArf the paid script prevents poofing? Or what is it exactly?

**Comment by Daemon53** [comment]
Yeah, the script we are using has an updated paid version

**Comment by shelby0341** [comment]
And that's just for player owned cars, rentals and business vehicles but not locals, yeah?

**Comment by IngleArf** [comment]
https://fivem-mods.tebex.io/package/4499436

**Comment by shelby0341** [comment]
Yeah I mean if it actually works I'll donate it
